web/src/modules/topic/components/Score/Score.styles.tsx

```diff
@@ -20,13 +20,61 @@ export const StyledButton = styled(Button, options)<StyledButtonProps>`
   border-radius: 50%;
 `;
 
-export const StyledPopper = styled(Popper)`
+export const ScorePopper = styled(Popper)`
+  display: flex;
+  position: absolute;
   border-radius: 50%;
   ${({ theme }) => css`
-    z-index: ${theme.zIndex.tooltip}; // even in front of toolbars
+    z-index: ${theme.zIndex.tooltip};
   `}
 `;
 
-export const StyledBox = styled(Box)`
-  pointer-events: none;
+interface BackdropProps {
+  isAnchored: boolean;
+}
+
+const backdropOptions = {
+  shouldForwardProp: (isAnchored: string) => !["isAnchored"].includes(isAnchored),
+};
+
+export const BackdropPopper = styled(Popper, backdropOptions)<BackdropProps>`
+  inset: 0;
+  ${({ theme }) => css`
+    z-index: ${theme.zIndex.tooltip};
+  `}
+  ${({ isAnchored }) => {
+    if (isAnchored)
+      return css`
+        background: black;
+        opacity: 0.5;
+      `;
+  }}
+`;
+
+interface CircleProps {
+  circleDiameter: number;
+}
+
+const circleOptions = {
+  shouldForwardProp: (circleDiameter: string) => !["circleDiameter"].includes(circleDiameter),
+};
+
+export const CircleDiv = styled("div", circleOptions)<CircleProps>`
+  /* Center content inside! */
+  height: ${({ circleDiameter }) => `${circleDiameter}px`};
+  width: ${({ circleDiameter }) => `${circleDiameter}px`};
```

Comment:
if the below two styles are not needed, i wonder if we need this div at all, since you created a parent div to this as well. it's probably worth trying to move these to CenterCircleDiv to see if we can avoid having this extra html element